iFixit/react-commerce
# Add collection display title frontend

closes #898 

This pr add a `displayTitle` field that allows to customise the h1 of collection pages without altering breadcrumbs, product-list children and other references to the collection default title.
The only other place in which the display title is impacting is inside collection meta-tags

### QA

1. Navigate to [Strapi](https://add-collection-display-title-frontend.govinor.com/admin)
2. Try to customize the `displayTitle` of a product-list 
3. Visit [Vercel preview](https://react-commerce-git-add-collection-display-title-frontend-ifixit.vercel.app/Parts)
4. Verify that the product list page renders correctly
5. Verify that the display title impacts only h1 and the meta-tags
